src/include/OpenImageIO/imagebufalgo.h

```diff
@@ -1925,6 +1925,7 @@ enum MakeTextureMode {
     MakeTxTexture, MakeTxShadow, MakeTxEnvLatl,
     MakeTxEnvLatlFromLightProbe,
     MakeTxBumpWithSlopes,
+    MakeTxCDF,
```

Comment:
I wonder if this is the right approach here at all -- it's not really a different texture type, it's an ordinary TxTexture, but with some extra metadata, right?

Perhaps a simpler way to express this is to treat it as TxTexture, but if the attributes contain either of the `maketx:cdfsigma*` attribute, then make_texture_impl knows to turn on this feature?

That would also let it fully work from `oiiotool ... -otex`, which I think your current implementation overlooked.

Reply:
Yes, you're right, and I think that makes prefect sense; I've moved them over to be all attributes now. I ended up with cdf as a on/off toggle attribute, so cdfbits and cdfsigma can have default values.